.azure/scripts/metadata_validator.py

```diff
@@ -48,4 +48,8 @@ def validate_thumbnail(thumbnail_src):
         return False
     return True
 
+def validate_image_url(image_url):
+    if not image_url.startswith('choreoanonymouspullable.azurecr.io'):
+        return False
+    return True
```

Comment:
```suggestion
return image_url.startswith('choreoanonymouspullable.azurecr.io')
```
We can simplify this logic.